Source/SwiftLintFramework/Rules/FileLengthRule.swift

```diff
@@ -2,8 +2,8 @@
 //  FileLengthRule.swift
 //  SwiftLint
 //
-//  Created by JP Simard on 2015-05-16.
-//  Copyright (c) 2015 Realm. All rights reserved.
+//  Created by JP Simard on 5/16/15.
```

Comment:
Some of these dates are now `mm/dd/yy` (like this one), others are now `dd/mm/yy`.